docs/configuration.md

```diff
@@ -153,6 +153,7 @@ Most of those properties are designed for performance tuning. Adjusting those nu
 * netty_client_threads: number of backend netty thread, default: number of logical processors available to the JVM.
 * default_workers_per_model: number of workers to create for each model that loaded at startup time, default: available GPUs in system or number of logical processors available to the JVM.
 * job_queue_size: number inference jobs that frontend will queue before backend can serve, default 100.
+* async_logging: enable asynchronous logging, default false.
```

Comment:
I would suggest to add some more description here regarding what is the logging behavior by default (that it is rotational) and why/when async could be useful instead.

Also, I think ./docs/logging.md needs to be updated to specify that multiple logging modes are supported with details on what each of them (rotational -vs- async) does.